src/test/java/org/opentripplanner/middleware/models/ItineraryTest.java

```diff
@@ -0,0 +1,97 @@
+package org.opentripplanner.middleware.models;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opentripplanner.middleware.otp.response.Itinerary;
+import org.opentripplanner.middleware.otp.response.Leg;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class ItineraryTest {
```

Comment:
Please add javadoc.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/91/commits/134a1c76a195169fd9c0a2c196409b3b4664f57c